specification/columns/chargeclass.md

```diff
@@ -32,6 +32,7 @@ Allowed values:
 | :--------- | :------------------------------------|
 | Regular    | Standard charges for services used or purchased. |
 | Correction  | Modification to one or more previous charges, like refunds and credit modifications. |
+| Percentage  | Denotes a charge that is billed NOT based on direct unit consumption but is based on a % cost of another service e.g. Support Costs |

```

Comment:
May a charge be Regular or Correction and also based on a Percentage?
Shouldn't Percentage be another column?